src/docker-in-docker/install.sh

```diff
@@ -332,13 +333,64 @@ fi
 
 usermod -aG docker ${USERNAME}
 
+# Function to fetch the latest version of the plugin
+get_latest_version() {
+    repo_url=$1
+    curl -s "$repo_url/latest" | jq -r '.tag_name'
+}
+
+# Function to fetch the version released prior to the latest version
+get_previous_version() {
+    latest_version="$1"
```

Comment:
Doesn't seem like this is being used, remove?

Reply:
removed 👍